recipes/recipes_emscripten/abseil-cpp/recipe.yaml

```diff
@@ -0,0 +1,46 @@
+context:
+  version: 20240116.2
+  name: abseil-split
```

Comment:
Since it's only one package, let's rename it to `libabseil`, the directory name as well.